src/javascript/app/pages/trade/markets.jsx

```diff
@@ -177,7 +177,7 @@ class Markets extends React.Component {
         let curr_market = null;
         Object.entries(market_nodes).forEach(([key, node]) => {
 
-            if (node && node.offsetParent && node.offsetTop - 41 <= position) {
+            if (node && node.offsetParent && node.offsetTop - 100 <= position) {
```

Comment:
can you add comment to describe what is 100 here, or create a named variable instead 🙏

Reply:
comment added